mnswaleh/sendIT
# bg-remove-header-161689979

#161689979 remove user header on admin page

#### What does this PR do?
Remove user header on update delivery page

#### Description of Task to be completed?
delete the header lines

#### How should this be manually tested?
If you open the update order on admin panel, you will not see the header with navigation buttons

#### What are the relevant pivotal tracker stories?
#161689979
